src/Apps/FeatureAKG/Components/Feature.tsx

```diff
@@ -0,0 +1,227 @@
+import { Box, Image, Sans, Serif } from "@artsy/palette"
+import { Feature_viewer } from "__generated__/Feature_viewer.graphql"
+import { FeaturedArticlesFragmentContainer as FeaturedArticles } from "Apps/FeatureAKG/Components/FeaturedArticles"
+import { FeaturedArtists } from "Apps/FeatureAKG/Components/FeaturedArtists"
+import { FeaturedRailsFragmentContainer as FeaturedRails } from "Apps/FeatureAKG/Components/FeaturedRails"
+import { FeaturedThisWeek } from "Apps/FeatureAKG/Components/FeaturedThisWeek"
+import { SelectedWorksFragmentContainer as SelectedWorks } from "Apps/FeatureAKG/Components/SelectedWorks"
+import { RouterLink } from "Artsy/Router/RouterLink"
+import { useSystemContext } from "Artsy/SystemContext"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
+import { crop, resize } from "Utils/resizer"
+
+interface FeatureProps {
+  viewer: Feature_viewer
+}
+
+const Feature: React.FC<FeatureProps> = props => {
+  const { injectedData } = useSystemContext()
+
+  if (injectedData === undefined) {
+    return null
+  }
+
+  const resizedUrl = resize(injectedData.hero_video_src, {
```

Comment:
`hero_video_src` - any reason this isn't `camelCase`?  (Haven't got to where the data comes from yet.)

Reply:
Oh yeah, all the stuff that's stored in s3 (via `JSONPage`) is snake_case-- only because the admin UI makes everything uppercase so without the `_` it gets confusing.

(It also makes it clear what's injected via force vs. fetched from metaphysics, which while developing was helpful).